.github/workflows/deploy-pipelines.yml

```diff
@@ -118,28 +121,50 @@ jobs:
             echo "next_version=${major}.${minor}.${patch}" >> $GITHUB_ENV
             echo "prerelease_enabled=false" >> $GITHUB_ENV
             echo "release_name=Release ${major}.${minor}.${patch} - $(date +'%Y-%m-%d')" >> $GITHUB_ENV
+
           elif [[ "${{ github.ref }}" == refs/heads/release-* ]]; then
-            # Pre-release - use same logic but append '-alpha/beta/rc.timestamp'
-            if [[ ${{ env.RELEASE_SEMVER }} == "major" ]]; then
-              major=$((major + 1))
-              minor=0
-              patch=0
-            elif [[ ${{ env.RELEASE_SEMVER }} == "minor" ]]; then
-              minor=$((minor + 1))
-              patch=0
+            # Branch di release: determinare se è prerelease o stabile
+            if [[ "${{ inputs.is_stable_release }}" == "true" ]]; then
+              # Trattare come una release stabile
+              if [[ ${{ env.RELEASE_SEMVER }} == "major" ]]; then
```

Comment:
can we put this if else block outside the upper if. it seems identical to the if else block of the case below